mobile/screens/UploadListScreen.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import { StyleSheet, Text, View } from 'react-native'
+
+import FileSelector from '../components/FileSelector'
+import UploadFileMutation from '../queries/UploadFileMutation'
+import UploadListQuery from '../queries/UploadListQuery'
+import UploadedList from '../components/UploadedList'
+
+class UploadListScreen extends React.Component {
+  render() {
+    return [
+      <UploadFileMutation key="uploader">
+        {execute => <FileSelector onSelected={execute} />}
+      </UploadFileMutation>,
+      <UploadListQuery key="uploaded-list">
+        {({ uploads }) => <UploadedList uploads={uploads} />}
+      </UploadListQuery>,
```

Comment:
The trailing comma must be a lint error. Keep in mind that you can use React fragments now (does this work in RN?):

```jsx
import { Fragment } from 'react'

export const Foo = () => (
  <Fragment>
    <div />
    <div />
  </Fragment>
)
```

Reply:
Maybe, i hadn't tried. :D